horizon/config.py

```diff
@@ -69,6 +69,12 @@ class SidecarConfig(Confi):
         10,
     )
 
+    OPA_CLIENT_QUERY_TIMEOUT = confi.float(
+        "OPA_CLIENT_QUERY_TIMEOUT",
+        5 * 60,  # This is also aiohttp's default timeout
```

Comment:
Default timeout should be much less for OPA.
I would say anything above 500ms should be a timeout.
WDYT?

Reply:
I think 500ms might cause us a lot of backlash in the support channels :)
I chose 5m to keep the default behavior we had before.
Let's compromise for 10secs?

I've been able to go above 2secs when added to OPA ~500k users 😋  (I think it was above 5secs as well)